src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -178,7 +178,8 @@ private void InitializeOutputFile(TOptions analyzeOptions, TContext context, Has
                                 analyzeOptions.Verbose,
                                 targets,
                                 analyzeOptions.ComputeTargetsHash,
-                                Prerelease)),
+                                Prerelease,
+                                null)),
```

Comment:
`invocationInfoTokensToRedact: null` (I really like the convention of using a named parameter whenever the value doesn't communicate the meaning, such as for literal values.)